django/curator/views.py

```diff
@@ -1,14 +1,17 @@
 from django.contrib.auth.decorators import permission_required
+from django.contrib import messages
+from django.shortcuts import get_object_or_404, redirect
 from django.http import HttpResponseRedirect, HttpResponseBadRequest
+from django.urls import reverse
 from django.views.decorators.http import require_POST
-from wagtail.contrib.modeladmin.helpers import AdminURLHelper
+from wagtail_modeladmin.helpers import AdminURLHelper
```

Comment:
should eventually migrate to Snippets or alternative https://github.com/comses/planning/issues/248